### DISCUSSION TITLE
Additional configuration required to show memory information.

### DISCUSSION THREAD

**Discussion by jgratero** [hardinfo2/hardinfo2#68] [Q&A] [answered]
![Screenshot_2024-08-20_14-21-31](https://github.com/user-attachments/assets/fdc68c73-5862-415e-8aac-220a55fc219c)

I wanted to verify the memory information, but all I get is this screen, without an indication of what I am supposed to do. I've looked for documentation related dmidecode, but I have found nothing related to the supposed configuration that I am supposed to set up at this point.

**Comment by hwspeedy** [comment, marked as answer]
1. Do you have dimm modules?
2. have you done what is written in the blue hint box, that you can see the top of in your screenshot?

The next release of hardinfo2 [see status #21] has a service to help us with DMI (Bios information) and SPD (DIMM eeprom) for memory modules. 

**Status**
The kernel group developers are doing a lot to clean up the SPD, which is kind of tricky as the communication to the little eeprom, that contains the DIMM info is over several different I2C/I3C SMBusses depending on Chipset and memory. Lots of setups are not supported - eg. servers with ECC memory etc. DDR5 requires kernel 6.11 and the decoding is not done yet.

![image](https://github.com/user-attachments/assets/1e478b26-df3a-4ce5-a467-099f6c6b3f42)
